concrete/src/Logging/Channels.php

```diff
@@ -62,7 +62,7 @@ class Channels
      *
      * @var string
      */
-    const CHANNEL_SITE_ORGANIZATION = 'site_organization';
+    const CHANNEL_PAGES = 'pages';
```

Comment:
What about keeping this constant (maybe marking it as `@deprecated`): that way we won't break 3rd party code that uses it...